docs/csharp/roslyn-sdk/syntax-visualizer.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Explore code with the syntax visualizer
+description: The syntax visualizer provides a visual tool to explore the models the .NET Compiler Platform SDK generates for code.
+author: billwagner
+ms.author: wiwagn
+ms.date: 03/07/2018
+ms.topic: conceptual
+ms.prod: .net
+ms.devlang: devlang-csharp
+ms.custom: mvc
+---
+# Explore code with the syntax visualizer
+
+This article provides an overview of the Syntax Visualizer tool that ships as part of the .NET Compiler Platform ("Roslyn") SDK. The Syntax Visualizer is a tool window that helps you inspect and explore syntax trees. It's an essential tool to understand the models for code you want to analyze. It's also a debugging aid when you develop your own applications using the .NET Compiler Platform (“Roslyn”) SDK. Open this tool as you create your first analyzers. The visualizer helps you understand the models used by the APIs.
+
+[!INCLUDE[interactive-note](~/includes/roslyn-installation.md)]
+
+Familiarize yourself with the concepts used in the .NET Compiler Platform SDK by reading the [overview](compiler-api-model.md) article. It provides an introduction to syntax trees, nodes, tokens, and trivia.
+
+## Syntax Visualizer
+
+The **Syntax Visualizer** enables inspection of the syntax tree for the C# or VB code file in the current active editor window inside the Visual Studio IDE. The visualizer can be launched by clicking on **View** –> **Other Windows** –> **Syntax Visualizer**.  You can also use the **Quick Launch** toolbar in the upper right corner. Type "syntax" and the command to open the **Syntax Visualizer** should appear.
+
+![Launching the Syntax Visualizer](media/fig1.png)
+
+This command opens the Syntax Visualizer as a floating tool window. If you don't have a code editor window open, the display is blank, as shown in the following figure. 
+
+![The Syntax Visualizer tool window](media/fig2.png)
+
+Dock this tool window at a convenient location inside Visual Studio, such as the left side. The Visualizer shows information about the current code file.
+
+Create a new project using the **File** –> **New Project** command. You can create either a VB or C# project. When Visual Studio opens the main code file for this project, the visualizer displays the syntax tree for it. You can open any existing C# / VB file in this Visual Studio instance, and the visualizer displays that file's syntax tree. If you have multiple code files open inside Visual Studio, the visualizer will always display the syntax tree for the currently active code file, the code file that has keyboard focus.
```

Comment:
Acrolinx will tell you to use the present tense where possible, i.e. "the visualizer displays the syntax tree for..."